x/oracle/types/denom.go

```diff
@@ -57,3 +58,16 @@ func (dl DenomList) ContainDenoms(d DenomList) bool {
 
 	return true
 }
+
+// GetRewardBand returns the reward band of a given asset in the DenomList.
+// It will return an error if it can not find it.
+func (dl DenomList) GetRewardBand(rbl RewardBandList) (sdk.Dec, error) {
```

Comment:
Can we make this a little more efficient by just passing in the denom as a param rather than looping through all the denoms in the denom list to find it?

Reply:
you actually found a big bug, so thank you @rbajollari < 3